PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-aad/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    implementation project(":azure-intellij-plugin-lib")
+
+    // as in the IntelliJ plugin's main build.gradle file
+    implementation 'com.microsoft.azuretools:azuretools-core:3.52.0-SNAPSHOT', {
```

Comment:
You should not depend on this module as we are about to discard it in near future.

Reply:
@wangmingliang-ms Understood, I've removed the dependency. If I need to use code of azuretools-core in the future, then I'll submit PRs for that.